packages/nodes-base/nodes/EmailSend/v2/send.operation.ts

```diff
@@ -223,11 +223,9 @@ export async function execute(this: IExecuteFunctions): Promise<INodeExecutionDa
 					});
 
 				for (const propertyName of attachmentProperties) {
-					if (!item.binary.hasOwnProperty(propertyName)) {
```

Comment:
this change makes this node consistent with other nodes, and now throws error when binary data is missing in input data, instead of silently ignoring it.
Should we mark this as a breaking change?

Reply:
I wouldn't create a new node version for this. If you specify attachements to send via email and they are not present it should be an error that should be fixed by the workflow builder. I'd suggest we leave it as it is.